app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/GeneDetailsMode.jsx

```diff
@@ -146,6 +161,11 @@ export default class GeneDetailsMode extends React.Component {
     const {
       ontology: { error },
     } = this.state;
+    const { geneName } = this.props;
```

Comment:
We generally don't log renders. Unless it's truly a special case.

You can often deduce renders from page views (which we log elsewhere) and api calls.

One reason why logging renders is tricky is that with React, it's best practice to assume that a render function can be called any number of times. Which is bad here because it means your log function can be called a bunch.

It might be better here to do a `Rails.logger.error` in the `fetch_ontology` controller function.